seed/models/building_file.py

```diff
@@ -45,6 +46,16 @@ class BuildingFile(models.Model):
         (HPXML, 'HPXML')
     )
 
+    BUILDINGSYNC_V1 = 1
+    BUILDINGSYNC_V2 = 2
+    BUILDINGSYNC_V2_ALT = 3
```

Comment:
what is the meaning of V2_Alternative? Why is this different than V2?

Reply:
See my comment on deprecating V2.